info-v4-resources.go

```diff
@@ -210,6 +230,94 @@ type ErasureSetResource struct {
 	DeleteMarkersCount uint64   `json:"deleteMarkersCount" msg:"dmc"`
 }
 
+// ClusterSummaryUsage contains storage usage statistics for the cluster
+type ClusterSummaryUsage struct {
+	TotalCapacity int64 `json:"totalCapacity" msg:"tc"`
+	Available     int64 `json:"available" msg:"av"`
+	DrivesUsage   int64 `json:"drivesUsage" msg:"du"`
+}
+
+// ClusterSummaryCount contains resource counts with status breakdown
+type ClusterSummaryCount struct {
+	Total   int `json:"total" msg:"t"`
+	Online  int `json:"online" msg:"on"`
+	Offline int `json:"offline" msg:"off"`
+	Healing int `json:"healing" msg:"hl"`
+}
+
+// DriveSummaryCount contains drive counts with status breakdown
+type DriveSummaryCount struct {
+	Total   int `json:"total" msg:"t"`
+	Online  int `json:"online" msg:"on"`
+	Offline int `json:"offline" msg:"off"`
+	Healing int `json:"healing" msg:"hl"`
+}
+
+// PoolDetails contains detailed configuration and statistics for a storage pool
+type PoolDetails struct {
+	TotalServers       int `json:"totalServers" msg:"ts"`
+	TotalObjects       int `json:"totalObjects" msg:"to"`
+	TotalDeleteMarkers int `json:"totalDeleteMarkers" msg:"tdm"`
+	TotalVersions      int `json:"totalVersions" msg:"tv"`
+	ErasureSets        int `json:"erasureSets" msg:"es"`
+	DrivesPerSet       int `json:"drivesPerSet" msg:"dps"`
+	Parity             int `json:"parity" msg:"p"`
+}
+
+// PoolSummary contains summary information for a storage pool including usage and drive statistics
+type PoolSummary struct {
+	Index   int                 `json:"index" msg:"idx"`
+	Usage   ClusterSummaryUsage `json:"usage" msg:"us"`
+	Drives  DriveSummaryCount   `json:"drives" msg:"drv"`
+	Details PoolDetails         `json:"details" msg:"dtls"`
+}
+
+// ClusterSummaryResponse contains a comprehensive summary of cluster resources and statistics
+type ClusterSummaryResponse struct {
+	Encryption   bool                `json:"encryption" msg:"enc"`
+	Version      string              `json:"version" msg:"ver"`
+	DeploymentID string              `json:"deploymentID" msg:"did"`
+	Region       string              `json:"region" msg:"reg"`
+	Domains      []string            `json:"domains" msg:"dom"`
+	Mode         string              `json:"mode" msg:"mod"`
+	Usage        ClusterSummaryUsage `json:"usage" msg:"us"`
+	Servers      ClusterSummaryCount `json:"servers" msg:"srv"`
+	Drives       ClusterSummaryCount `json:"drives" msg:"drv"`
```

Comment:
should `Drives` be `DriveSummaryCount` ? and then maybe we can have the `ClusterSummaryCount` no longer need Healing field for servers?